api/v1alpha2/inferencepool_types.go

```diff
@@ -182,7 +182,7 @@ type InferencePoolStatus struct {
 // PoolStatus defines the observed state of InferencePool from a Gateway.
 type PoolStatus struct {
 	// GatewayRef indicates the gateway that observed state of InferencePool.
-	GatewayRef corev1.ObjectReference `json:"parentRef"`
+	GatewayRef v1alpha1.ParentGatewayReference `json:"parentRef"`
```

Comment:
I just import it from GW api instead of coping it as you mentioned in #1149. Let me know whether it works for you @robscott